docs/en_US/Compression/Pruner.rst

```diff
@@ -722,3 +723,134 @@ User configuration for Sensitivity Pruner
+             'dummy_input': dummy_input,
+             'trainer': trainer,
+             'optimizer': optimizer
+             }
+    config_list = [{
+        'sparsity': 0.5,
+        'op_types': ["Linear"]
+    }]
+   pruner = TransformerHeadPruner(model, config_list, **kwargs)
+   pruner.compress()
+
+Usage 2: same effect as usage 1, the only change is passing names to the pruner instead of dummy input (PyTorch code)
+
+.. code-block:: python
+
+   from nni.algorithms.compression.pytorch.pruning import TransformerHeadPruner
+   attention_name_groups = list(zip(['encoder.layer.{}.attention.self.query'.format(i) for i in range(12)],
+                                    ['encoder.layer.{}.attention.self.key'.format(i) for i in range(12)],
+                                    ['encoder.layer.{}.attention.self.value'.format(i) for i in range(12)],
+                                    ['encoder.layer.{}.attention.output.dense'.format(i) for i in range(12)]))
+   kwargs = {'ranking_criterion': "l1_weight",
+             'global_sort': False,
+             'num_iterations': 1,
+             'epochs_per_iteration': 1,    # this is ignored when num_iterations = 1
+             'head_hidden_dim': 64,
+             'attention_name_groups': attention_name_groups,
+             'trainer': trainer,
+             'optimizer': optimizer
+             }
+    config_list = [{
+        'sparsity': 0.5,
+        'op_types': ["Linear"]
+    }]
+   pruner = TransformerHeadPruner(model, config_list, **kwargs)
+   pruner.compress()
+
+Usage 3: one-shot pruning, setting different sparsity for different layers (PyTorch code)
+
+.. code-block:: python
+
+   from nni.algorithms.compression.pytorch.pruning import TransformerHeadPruner
+   attention_name_groups = list(zip(['encoder.layer.{}.attention.self.query'.format(i) for i in range(12)],
+                                    ['encoder.layer.{}.attention.self.key'.format(i) for i in range(12)],
+                                    ['encoder.layer.{}.attention.self.value'.format(i) for i in range(12)],
+                                    ['encoder.layer.{}.attention.output.dense'.format(i) for i in range(12)]))
+   kwargs = {'ranking_criterion': "l1_weight",
+             'global_sort': False,
+             'num_iterations': 1,
+             'epochs_per_iteration': 1,    # this is ignored when num_iterations = 1
+             'head_hidden_dim': 64,
+             'attention_name_groups': attention_name_groups,     # can change to dummy_input here
+             'trainer': trainer,
+             'optimizer': optimizer
+             }
+    config_list = [{
+        'sparsity': 0.5,
+        'op_types': ["Linear"],
+        'op_names': [x for layer in attention_name_groups[:6] for x in layer]      # first six layers
+    },
+    {
+        'sparsity': 0.25,
+        'op_types': ["Linear"],
+        'op_names': [x for layer in attention_name_groups[:6] for x in layer]      # last six layers
```

Comment:
this is for the last six layers? it is the same as line 839

Reply:
fixed